### PR TITLE
Updating Goodies.toml

### PR DESCRIPTION
**Pull request by h12guerrero**
Adding a ReadMe Generator

### CODE REVIEW

**Comment by D3Portillo** [comment]
Thank you @SoldierHD  :' 3

**Comment by D3Portillo** [comment]
Will update `type` prop into "tool" , to match [available types](https://github.com/D3Portillo/goodies4devs/blob/master/src/types.js) : ) .

File: goodies.toml [hunk 1]
```diff
@@ -205,3 +205,9 @@ website = "https://www.classcentral.com/"
 contributor = "@chrisft25"
 type = "course"
 money = "freemium"
+
+["GitHub Profile ReadMe Generator"]
+description = "ReadMe Generator for GitHub Profile"
+website = "https://github.com/rahuldkjain/github-profile-readme-generator"
+contributor = "@SoldierHD"
+type = "Github"
```
Comment (D3Portillo) [inline goodies.toml]: Can you please update to: `type = "tool"` ?

**Review by D3Portillo — CHANGES_REQUESTED** [review]
Un gusto Héctor, solo si porfavor puedes actualizar el tipo de goodie a "tool"

File: goodies.toml (hunk 1 above)
Comment (h12guerrero) [reply goodies.toml]: Okay

Comment (h12guerrero) [reply goodies.toml]: Updated!

**Comment by D3Portillo** [comment]
:heart:

**Review by D3Portillo — APPROVED** [review]

**Pull request merged**